src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+        <img src="images/Dark_theme_01_new.png" alt="Do not apply components that are darker than the background unless using high-contrast mode.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Dark theme sample](images/Dark_theme_sample_revised.gif)
+
+_Components with common colors across both dark UI backgrounds_
+
+### High contrast
+
+In some cases, it is helpful to apply light components to dark backgrounds or light components to dark backgrounds. This technique is useful to focus attention or create visual tension.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_01.png" alt="High contrast example in a light theme.">    
+    </do-dont-example>
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_02.png" alt="High contrast example in a dark theme.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+## Interaction states
+
+In addition to the enabled-state tokens above, the five other primary interaction states have been defined and tokenized for each default theme. For values between Black and 70, interaction gets lighter, and for values between 60 and White, interaction gets darker.
+
+### Hover
+Hover states have their own tokens. In the IBM default themes, these colors fall outside of the core palette steps and act as "half steps" between two adjacent colors. The White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
+
+### Active
+The active state is two full steps lighter for values between 100 and 70 and two full steps darker for values between 60 and 10. For example, the Blue 60 active state is Blue 80. The exception is that White shares the same active state as Gray 10, and Black shares the same active state as Gray 100. 
+
+### Selected
+The selected state is one full step lighter for values between 100 and 70, and one full step darker for values between 60 and 10. For example, the Blue 60 selected state is Blue 70. The exception is that White shares the same selected state as Gray 10, and Black shares the same selected state as Gray 100. 
+
+### Focus
+The focus state usually appears as a Blue 60 border in the light theme and as a White border in the dark theme.
+
+### Disabled
+Disabled components are unavailable for interaction, so they don't receive hover or focus and are not subject to WC3 contrast compliance standards. For the dark themes, disabled values are two full steps lighter than their respective background. For light themes, they are two full steps darker. The exception is that the White theme disabled value is Gray 30.     
+
+![Interactive states light theme](images/Interactive_states_light_new.png)
+
+_Light theme interactive tokens_
+
+
+![Interactive states dark theme](images/Interactive_states_dark_new.png)
+
+_Dark theme interactive tokens_
+
+
+## Digital accessibility
```

Comment:
Again I don't like the word "digital" and I know Jeoff isn't a fan of it either. Can't this just say "Color accessibility"